sky/packages/sky/lib/src/rendering/paragraph.dart

```diff
@@ -80,15 +87,15 @@ class RenderParagraph extends RenderBox {
 
   double computeDistanceToActualBaseline(TextBaseline baseline) {
     assert(!needsLayout);
-    _layout(constraints);
-    return _textPainter.computeDistanceToActualBaseline(baseline);
+    layoutText(constraints);
+    return textPainter.computeDistanceToActualBaseline(baseline);
   }
 
   void performLayout() {
-    _layout(constraints);
+    layoutText(constraints);
     // _paragraphPainter.width always expands to fill, use maxContentWidth instead.
```

Comment:
replace this comment with "We use textPainter.maxContentWidth here, rather that textPainter.width, because the latter is the width that it used to wrap the text, whereas the former is the actual width of the text."